packages/playground/src/weblets/tf_contracts_list.vue

```diff
@@ -306,6 +315,18 @@ function getNodeStateColor(state: NodeStatus): string {
       return "warning";
   }
 }
+
+function getTotalCost(contracts: NormalizedContract[]) {
```

Comment:
I believe we should introduce the raw field instead along with the consumption string, so field `tftPerHour` on the normalized contract object, and drop the regex part